geo.txt

```diff
@@ -87,9 +87,9 @@ spectroscopy and ab initio calculations. J Chem Phys 147(23):234308/1-234308/8
 https://doi.org/10.1063/1.5008744
 # Date first added: 2023-07-05 (by Sichao Xiong)
 
-Ag   I   2.5375(3)
-Ag...N   2.180(1)
-Ag   N H 110.86(5)
+Ag   I     2.5375(3)
+Ag...NH3   2.180(1)
```

Comment:
I wrote a comment here a moment ago but can't find what I wrote anymore. I was saying that Ag...NH3 suggests that you're providing the length between Ag and the center-of-mass of NH3, when really the book looks like it's denoting the distance between Ag and N.

Reply:
See above